src/http/app.ts

```diff
@@ -283,3 +283,59 @@ app.post(
     matchesHandler(req, res, 201, true);
   }
 );
+
+app.get("/chains/:chainId/rounds/:roundId/matches.csv", async (req, res) => {
```

Comment:
We shouldn't duplicate the matches handler since it's exactly the same as the other one, the only difference is the output format. We can extract it to it's own function and then from the return value, encode the data in JSON or CSV.  You can also create a CSV from an array of objects: https://www.npmjs.com/package/csv-writer#createobjectcsvwriterparams

Reply:
As I've written before: we don't need this endpoint. We already have the match data on the frontend, and the code to transform it from json to csv is already implemented, working and tested. Has been for more than a week. I really don't know the reason for reopening this.

Reply:
@vacekj that makes a lot of sense, sorry I missed your previous message on this! then to conclude, unless we can think of another reason we need this endpoint in the server I would go with your solution for the reasons you outlined. @gravityblast is this ok with you?

Reply:
yes we decided in the past that this PR was a duplicate of the other where we create the CSV in the fronend, we can close it. @vacekj did we already merge the other one?

Reply:
not yet, it's this one, we should review it: https://github.com/gitcoinco/grants-stack/pull/1672

Reply:
It's part of the new PR i raised for #1475

Reply:
closing this then thank you for your input everyone